src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSPathogenReferenceTaxonProperties.java

```diff
@@ -11,10 +14,34 @@
     public String name = null;
     public String rank = null;
     public String parentTaxId = null;
-    public long length = 0;
-    public final Set<String> accessions;
+    private long length = 0;
```

Comment:
Isn't quite possible we usually read in the taxonomic ID long before we get its name, in parseNamesFile(). I added get/set methods for PSPathogenReferenceTaxonProperties.